products/compute/api.yaml

```diff
@@ -11352,83 +11356,154 @@ objects:
             - !ruby/object:Api::Type::String
               name: 'pathMatcher'
               required: true
               description: |
-                The name of the PathMatcher to use to match the path portion of
-                the URL if the hostRule matches the URL's host portion.
-      - !ruby/object:Api::Type::Integer
-        name: 'id'
-        description: 'The unique identifier for the resource.'
-        output: true
-      - !ruby/object:Api::Type::Fingerprint
-        name: 'fingerprint'
-        description: |
-          Fingerprint of this resource. This field is used internally during
-          updates of this resource.
+                The name of the PathMatcher to use to match the path portion of the URL if the
+                hostRule matches the URL's host portion.
       - !ruby/object:Api::Type::String
         name: 'name'
         required: true
         input: true
         description: |
-          Name of the resource. Provided by the client when the resource is
-          created. The name must be 1-63 characters long, and comply with
-          RFC1035. Specifically, the name must be 1-63 characters long and match
-          the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the
-          first character must be a lowercase letter, and all following
-          characters must be a dash, lowercase letter, or digit, except the last
-          character, which cannot be a dash.
+          Name of the resource. Provided by the client when the resource is created. The
+          name must be 1-63 characters long, and comply with RFC1035. Specifically, the
+          name must be 1-63 characters long and match the regular expression
+          `[a-z]([-a-z0-9]*[a-z0-9])?` which means the first character must be a lowercase
+          letter, and all following characters must be a dash, lowercase letter, or digit,
+          except the last character, which cannot be a dash.
       - !ruby/object:Api::Type::Array
         name: 'pathMatchers'
-        description: 'The list of named PathMatchers to use against the URL.'
+        description: |
+          The list of named PathMatchers to use against the URL.
         item_type: !ruby/object:Api::Type::NestedObject
           properties:
             - !ruby/object:Api::Type::ResourceRef
               name: 'defaultService'
-              required: true
               resource: 'BackendService'
               imports: 'selfLink'
               description: |
-                A reference to a BackendService resource. This will be used if
-                none of the pathRules defined by this PathMatcher is matched by
-                the URL's path portion.
+                The BackendService resource. This will be used if
+                none of the pathRules or routeRules defined by this PathMatcher are matched. For
+                example, the following are all valid URLs to a BackendService resource:   - http
```

Comment:
These docs are nominally formatted markdown, can you separate the list items line-by-line?